extensions/landing-zone/variables.tf

```diff
@@ -0,0 +1,58 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "APIkey associated with the account to use"
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "Region to provision all resources created by this solution"
+  default     = "au-syd"
```

Comment:
I would remove the default region to make the region mandatory (as it is the region of the cluster where it is going to be deployed), I would set `nullable = false`